### DISCUSSION TITLE
Adding points web page include form fields several times when Javascript is disabled

### DISCUSSION THREAD

**Discussion by petterreinholdtsen** [tosdr/edit.tosdr.org#1037] [Ideas]
When visiting <URL: https://edit.tosdr.org/services/514/points/new > with a browser where JavaScript is blocked, the status and source fields are shown twice.  It would be better if they only showed up once.  The status setting is also visible, which is not visible when JavaScript is enabled.

**Comment by michielbdejong** [comment]
Interesting! We don't officially support disabling JavaScript, and there are other parts of edit.tosdr.org (especially the annotate view which will soon take a more central place in the reviewing process) that will not work without JavaScript.

But if someone has a fix for this that works both with and without JavaScript, then we can merge it! :)
